### ISSUE TITLE
bar-descriptor.xml - Momentics interoperability

### ISSUE THREAD

**Issue by tomgregar** [phofman/vs-plugin#43]
I tried to open project generated in VS plugin (VSP) in Momentics - to design qml there. Some warnings occured in Bar-descriptor.xml:
- generated bar-descriptor.xml from VSP uses element splashscreen instead of SplashScreens (the former is deprecated)
- Momentics does not like element transparent in initialWindow (even if this element is optional, not deprecated, in public DTD). 
  Is it possible to change generated XML?

One bigger issue is related with this - Momentics cannot open VS project as is, it only can import "existing code as Blackberry c/c++ makefile project". This works, qml is editable, you can save it and open in VS etc. That process adds Momentics .project and .cproject files. BUT - it also overwrite config.pri. And something in that import  automatically adds Build Configurations _Device-Coverage_, _Simulator-Profile_ and _Simulator-Coverage_ without entry-point causing error if someone try to edit bar-descriptor.xml/Build configurations in Momentics UI. The good thing is that project remains buildable in VS.

It would be nice if VSP - when creating/updating project - could generate also Momentics project metadata - so it will be openable in Momentics without import-based issues.

**Comment by phofman** [comment]
Yes, I agree the bar-descriptor editor needs a bigger update (read: total rewrite). It has lots of other issues not mentioned here: there are only few sub-editors comparing to ones from Momentics, it enforces its own attributes order, removes comments (reformats the document) and UI could be improved.

But generating Momentics metadata is not the path I would like to go. I don't want to be asked to launch Momentics at all ;)
